raspberry2arduino.py

```diff
@@ -4,33 +4,30 @@
 GPIO.setmode(GPIO.BOARD)
 GPIO.setup(10, GPIO.IN,pull_up_down=GPIO.PUD_DOWN)
 GPIO.setup(8, GPIO.OUT )
-drinkAt=0
-drinkTo=2
+drinkAt=0                          #the drink the machine is currently at
+drinkTo=2                          #The drink the machine has to go to
+amountOfSpots=12
+timePerBucket=0.5
+
+distance1= drinkTo-drinkAt         #the distance counterclockwise
```

Comment:
this is clockwise not counterclockwise, also note that the machine starts at bottle 0 not 1